doc/examples/reconst_ivim.py

```diff
@@ -0,0 +1,211 @@
+"""
+We get an IVIM dataset using Dipy's data fetcher `read_ivim`.
+This dataset was acquired with 21 b-values in 3 different directions.
+Volumes corresponding to different directions were registered to each
+other, and averaged across directions. Thus, this dataset has 4 dimensions,
+with the length of the last dimension corresponding to the number
+of b-values.
+"""
+
+img, gtab = read_ivim()
+
+"""
+img contains a nibabel Nifti1Image object(with the data)
+and gtab contains a GradientTable object(information about
+the gradients e.g. b-values and b-vectors). We get the
+data from img using `read_data`.
+"""
+
+data = img.get_data()
+print('data.shape (%d, %d, %d, %d)' % data.shape)
+
+"""
+The data has 54 slices, with 256-by-256 voxels in each slice.
+The fourth dimension corresponds to the b-values in the gtab.
+Let us visualize the data by taking a slice midway(z=27) at bvalue = 0.
+"""
+
+z = 33
+b = 20
+
+plt.imshow(data[:, :, z, b].T, origin='lower', cmap='gray')
+plt.axhline(y=100)
+plt.axvline(x=170)
+plt.savefig("ivim_data_slice.png")
+plt.close()
+
+"""
+The marked point in the image shows a section containing cerebral spinal fluid (CSF)
+so it should have a very high f and D*, the area between the right and left is white
+matter so that should be lower, and the region on the right is gray matter and CSF.
+That should give us some contrast to see the values varying across the regions.
+"""
+
+x1, x2 = 160, 180
+y1, y2 = 90, 110
+data_slice = data[x1:x2, y1:y2, z, :]
+
+plt.imshow(data[x1:x2, y1:y2, z, b].T, origin='lower', cmap="gray")
+plt.savefig("CSF_slice.png")
+plt.close()
+
+"""
+Now that we have prepared the datasets we can go forward with
+the ivim fit. Instead of fitting the entire volume, we focus on a
+small section of the slice, to fit the IVIM model. First, we instantiate
+the Ivim model. Here, we use a two-stage approach: first, a tensor
+is fit to the data, and then this tensor as the initial starting point
+for the non-linear fit of IVIM parameters. All initializations are
+passed with the `IvimModel` such `split_b`. If you are using
+Scipy 0.17, you can also set bounds by setting
+`bounds=([0., 0., 0., 0.], [np.inf, 1., 1., 1.]))` while initializing
+the IvimModel. It is recommeded that you upgrade to Scipy 0.17 since
+the fitting results might at times return values which do not make sense
+physically. (Negative f values or D*)
+"""
+
+ivimmodel = IvimModel(gtab)
+
+"""
+To fit the model, call the `fit_method` and pass the data for fitting.
+"""
+
+ivimfit = ivimmodel.fit(data_slice)
+
+"""
+The fit method creates a IvimFit object which contains the
+fitting parameters of the model the fit parameters of the model.
+These are accessible through the model_parameters attribute of the
+IvimFit object. Parameters are arranged as a 4D array, corresponding
+to the spatial dimensions of the data, and the last dimension (of length 4)
+corresponding to the model parameters according to the following
+order: S0, f, D* and D.
```

Comment:
Use math-dollar latex script as you did above after the equations for the parameters.